.gersemirc

```diff
@@ -0,0 +1,12 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/BlankSpruce/gersemi/master/gersemi/configuration.schema.json
+
+cache: true
+color: false
+definitions: []
+indent: 2
+line_length: 120
+list_expansion: favour-inlining
+quiet: false
+unsafe: false
+warn_about_unknown_commands: false
+workers: 16
```

Comment:
Maybe we should avoid it? The default is: `[default: number of CPUs on this system]`